examples/module/chardev/chardev.c

```diff
@@ -121,14 +122,9 @@ static int module_uninitialize(FAR void *arg)
  *
  ****************************************************************************/
 
-int module_initialize(FAR struct mod_info_s *modinfo)
+__attribute__((constructor))
```

Comment:
See other comments with regard to the explicit use of GCC attributes in common code.